src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -68,10 +68,11 @@ class ViewClippingWindow final : public Window
 {
 private:
     CoordsXY _selectionStart;
-    CoordsXY _previousClipSelectionA;
-    CoordsXY _previousClipSelectionB;
+    CoordsXY _previousClipSelectionA = { 0, 0 };
+    CoordsXY _previousClipSelectionB = gMapSize.ToCoordsXY();
```

Comment:
This creates an initialization order dependency. If someone were to open this window with a hotkey while a map is being loaded, `gMapSize` may not yet be initialized (it may now, but you can't depend on it). The `OnOpen` function should be used for this instead.

Reply:
Doesn’t that have the same problem?

Reply:
The difference is that `OnOpen` is called explicitly within the event stack, while the constructor is called as soon as the window is constructed. It's saver to assume the events are called in a proper order.